README.md

```diff
@@ -15,6 +15,9 @@ Find census events and resources near you.
 Instead of re-creating a calendar tool, we're leveraging a fantastic and proven
 one: Google Calendar.
 
+## Places API:
+(Reference)[https://developers.google.com/maps/documentation/javascript/places-autocomplete]
+(Create Credentials)[https://console.cloud.google.com/apis/credentials?project=census-events-258405&supportedpurview=project]
```

Comment:
Oh, just noticed this up here. I missed it when I looked at the Maps API, you should move them together.